cedar-wasm/src/validator.rs

```diff
@@ -22,10 +22,9 @@ use tsify::Tsify;
 use wasm_bindgen::prelude::*;
 
 #[derive(Tsify, Debug, Serialize, Deserialize)]
-#[serde(untagged)]
+#[serde(tag = "type", rename_all = "camelCase")]
```

Comment:
waiting on your feedback on my reply on the issue before we accept this change, for consistency's sake

Reply:
Thanks, I've added [some feedback here](https://github.com/cedar-policy/cedar/issues/806#issuecomment-2071916686). What I will do is revert the change in this PR because it makes the result types inconsisten, I can raise a separate PR to adjust _all_ of the TypeScript result types together depending on the outcome on the issue discussion.